tree/ntuple/inc/ROOT/RFieldBase.hxx

```diff
@@ -713,9 +715,13 @@ public:
    RValue &operator=(RValue &&other) = default;
    ~RValue() = default;
 
+private:
    std::size_t Append() { return fField->Append(fObjPtr.get()); }
+
+public:
    void Read(ROOT::NTupleSize_t globalIndex) { fField->Read(globalIndex, fObjPtr.get()); }
    void Read(RNTupleLocalIndex localIndex) { fField->Read(localIndex, fObjPtr.get()); }
```

Comment:
@jblomer we discussed making the `Read` methods `private` as well. This would be asymmetric to `RBulkValues`, so in the end I decided against it (for the moment)